src/ray/core_worker/core_worker.cc

```diff
@@ -1987,8 +1987,61 @@ void CoreWorker::YieldCurrentFiber(FiberEvent &event) {
   event.Wait();
 }
 
+void CoreWorker::PlasmaCallback(SetResultCallback success,
+                                std::shared_ptr<RayObject> ray_object, ObjectID object_id,
+                                void *py_future) {
+  std::vector<std::shared_ptr<RayObject>> vec;
+  if (Get(std::vector<ObjectID>{object_id}, 0, &vec).ok() && vec.size() > 0) {
+    return success(vec.front(), object_id, py_future);
+  }
```

Comment:
what's this case for?

Reply:
Any case where the object is immediately available, but not in Memory. Calling before subscribe avoids filling subscribe with unnecessary object ids.